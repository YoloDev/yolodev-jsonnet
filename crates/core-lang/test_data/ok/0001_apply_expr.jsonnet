local foo(a, b) = a + b + c,
      c = 10;

foo(c, b = 5)
