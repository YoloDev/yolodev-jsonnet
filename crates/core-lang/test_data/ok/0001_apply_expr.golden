local foo#2 =
        function(a#4 =
            error "Parameter \'a\' not bound",
          b#5 =
            error "Parameter \'b\' not bound",
        )
          ((a#4 + b#5) + c#3),
      c#3 = 10.0;
foo#2(c#3, b = 5.0)
